starters/vue3-xstate-css/src/machines/greetMachine.ts

```diff
@@ -0,0 +1,55 @@
+import { createMachine, assign } from 'xstate';
+import { fetchGreeting } from '../utils/fetchGreeting';
+
+export const greetMachine = (query: string) => {
+  return createMachine(
+    {
+      id: 'greet',
+      predictableActionArguments: true,
+      schema: {
+        context: {} as { query: string; message: string; error: string },
+        services: {} as {
+          callFetchMessage: {
+            data: { result: string };
+          };
+        },
+      },
+      initial: 'loading',
+      context: {
+        query,
+        message: '',
+        error: '',
+      },
+      states: {
+        loading: {
+          invoke: {
```

Comment:
When I run `yarn run type-check ` I receive this error message 

<img width="1606" alt="Screenshot 2023-04-05 at 12 13 36 PM" src="https://user-images.githubusercontent.com/67210629/230181773-c3f000d0-de05-4eb6-96b8-89479e3e2d32.png">

Reply:
I may need a comment here for this, that's a good point. This is exactly how the XState docs have it and from all I could find it should be supported, but I couldn't ever get this issue to clear. I'm hoping I can eventually share this kit and ask in their Discord about it, maybe I have something weird in my types. But I can add a comment for now to explain that.